docs/azure_arc_jumpstart/azure_arc_k8s/day2/multi_distributions/gitops/_index.md

```diff
@@ -0,0 +1,142 @@
+---
+type: docs
+title: "Managing Calico Network Policy using GitOps in an Azure Arc connected cluster"
```

Comment:
Please update to Azure Arc-enabled Kubernetes cluster

Reply:
done